resources/agent.rb

```diff
@@ -23,6 +23,8 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+require 'yaml'
```

Comment:
@derekgroh Do you know if this fixes #104 ? Seems like it might

Reply:
Only encountered it on windows platforms, but sounds like I should run back through most of the suites and confirm.

It would need to be added it to `resources/backend.rb`  as well.

Reply:
Not a major issue since it's not the main scope of this PR, I can take a look at it later. Was just curious.